zerver/webhooks/github/view.py

```diff
@@ -587,6 +599,12 @@ def get_event(request: HttpRequest, payload: Dict[str, Any], branches: Optional[
         if payload['check_run']['status'] != 'completed':
             return None
         return event
+    elif event == "team":
+        action = payload["action"]
+        if action == "edited":
+            return "team"
+        assert action in IGNORED_TEAM_ACTIONS
```

Comment:
I don't think we want an assertion here; they may add new ones in the future, and we want to raise the much less visible `UnexpectedWebhookEventType` for that case (our tooling knows that's a low-priority exception, logs it specially, etc.).

Reply:
I think the right fix is probably `elif event == "team" and action in IGNORED_TEAM_ACTIONS"?

Reply:
Phrased it like this now:

~~~ py
602     elif event == "team":
603         action = payload["action"]
604         if action == "edited":
605             return "team"
606         if action in IGNORED_TEAM_ACTIONS:
607             # no need to spam our logs, we just haven't implemented it yet
608             return None
609         else:
610             # this means GH has actually added new actions since September 2020,
611             # so it's a bit more cause for alarm
612             raise UnexpectedWebhookEventType("GitHub", f"unexpected team action {action}")